packages/app/src/app/pages/Sandbox/Editor/Workspace/Dependencies/index.tsx

```diff
@@ -32,26 +32,27 @@ export const Dependencies: FunctionComponent = () => {
     resource => !resource.includes('fonts.googleapis.com/css')
   );
 
-  if (!parsedConfigurations.package) {
+  if (!parsedConfigurations?.package) {
     return <ErrorMessage>Unable to find package.json</ErrorMessage>;
   }
 
-  const { parsed, error } = parsedConfigurations.package;
+  const { error, parsed } = parsedConfigurations.package;
```

Comment:
I think we need to add a default value to `parsed` here. Otherwise, it'll continue to break because it may be `undefined` when destructuring on line 46.

Reply:
No we don't need to, since we now for sure `error` is undefined on line 46, since if it wasn't we would've returned on line 41.